k8s/charts/seaweedfs/templates/shared/networkpolicy.yaml

```
@@ -0,0 +1,231 @@
+       master and filer, the resize hook shells out to kubectl.
+
+       The bucket hook is post-install, and Helm applies the release manifest
+       before post-install hooks run, so a plain policy is already in place by
+       then. The resize hook is pre-install weight 0, which runs before the
+       manifest is applied at all, so its policy has to be a pre-install hook
+       itself, at a weight below the Job's. */}}
+{{- /* Same condition as the Job in post-install-bucket-hook.yaml: an S3
+       endpoint plus buckets to create. Keep the two in step. */}}
+{{- $bucketHook := false }}
+{{- if .Values.allInOne.enabled }}
+{{-   $bucketHook = and .Values.allInOne.s3.enabled .Values.allInOne.s3.createBuckets }}
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)